File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraLayout;

namespace radialMenu
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void simpleButton1_Click(object sender, EventArgs e)
        {
            LayoutControlItem item = new LayoutControlItem();
            item.SizeConstraintsType = SizeConstraintsType.Custom;
            item.TextVisible = false;
            item.Control = layoutControl1;
            layoutControlGroup3.AddItem(item);
        }
    }
}

File: VB/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraLayout

Namespace radialMenu
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
			Dim item As New LayoutControlItem()
			item.SizeConstraintsType = SizeConstraintsType.Custom
			item.TextVisible = False
			item.Control = layoutControl1
			layoutControlGroup3.AddItem(item)
		End Sub
	End Class
End Namespace
